packages/nextjs/app/page.tsx

```diff
@@ -1,3 +1,6 @@
+"use client";
+
+import { useState } from "react";
```

Comment:
I messed up when I resolved conflicts, Can you remove this "useState" declaration and then we can merge.

Reply:
Resolved